coding-concepts.js

```diff
@@ -16,44 +16,50 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2022"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 9
+// b) Verify and explain: 10. Under String Methods and Properties in the syllabus, the function will count the characters in the string. There was nothing annotated in the function regarding the index, that was my error. I counted the characters starting at zero index but there were no brackets.
 
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "o"
+// b) Verify and explain: "o". Referencing the Index/ Accessing Elements section in the syllabus under JS Arrays. The function is asking to return the value at index 4. The index is defined by using the square brackets.
 
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "Ruby"
+// b) Verify and explain: "Ruby". Referencing the Index/ Accessing Elements section in the syllabus under JS Arrays. The function is asking to return the index variable, which is defined as 1. The zero indexing of the values results in "Ruby" being the value at 1 index. 
 
 
 // --------------------4) What will this log?
 
-const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+//const weekendDays = ["saturday", "sunday"] 
+//console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+
+// a) Your answer: An error would occur because either saturday or sunday needs to be entered into the parenthesis. 
+// b) Verify and explain: "TypeError: weekendDays.toUpperCase is not a function". weekendDays was not defined as a function. The parameters for "saturday" and "sunday" should be something different ie [dayOfWeek], since those inputs might be different every time the function is run. After the parameters are redefined, then a function can be written using string interpolation to return what ever day of the week is specified in the function.
```

Comment:
It is not necessarily the array itself that is causing the error here, it is the fact that .toUpperCase() is being ran on an array when it is only meant to work on strings.  To capitalize both values in the array, we would need to use iteration like a for loop.